setup.py

```diff
@@ -197,15 +196,6 @@ def build(self):
 with open(config_path, encoding='utf-8') as f:
     version = f.readline().strip()
 
-try:
```

Comment:
if you could enable this expanded version when self.debug is true that would be helpful for my use case